pkg/controller/nodes/task/taskexec_context.go

```diff
@@ -64,6 +64,14 @@ func (t taskExecutionMetadata) GetMaxAttempts() uint32 {
 	return t.maxAttempts
 }
 
+func (t taskExecutionMetadata) GetSecurityContext() core.SecurityContext {
```

Comment:
so @anandswaminathan is working on plumbing this all the way through. But I guess we can keep this, when Anand rebases, he can delete it

Reply:
sounds good!